reorder_python_imports.py

```diff
@@ -468,6 +468,12 @@ def _fix_file(
     'typing_extensions=typing:get_origin',
     'typing_extensions=typing:is_typeddict',
 ))
+REPLACES[(3, 11)].update((
+    'typing_extensions=typing:NamedTuple',
+    'typing_extensions=typing:TypedDict',
+    'typing_extensions=typing:final',
+    'typing_extensions=typing:overload',
+))
```

Comment:
like the other PR -- can you remove this from the output until 3.11 final has landed

Reply:
Done.